src/components/application_manager/test/commands/mobile/add_command_request_test.cc

```diff
@@ -229,6 +249,97 @@ TEST_F(AddCommandRequestTest, OnEvent_VR_UNSUPPORTED_RESOURCE) {
   }
 }
 
+TEST_F(AddCommandRequestTest, OnEvent_UI_AddCommand_UNSUPPORTED_RESOURCE) {
```

Comment:
@okozlovlux I propose write not `AddCommand` but HmiSendSuccess, it will show what actually this test checks